src/components/Notice/Header.tsx

```diff
@@ -0,0 +1,43 @@
+import { useNavigate } from 'react-router-dom';
+
+import styled from 'styled-components';
+
+type HeaderProps = {
+    children: React.ReactNode;
+    shadow: 'true' | 'false';
+}
+
+const HeaderContainer = styled.div <{ shadow: string }>`
+display: flex;
+align-items: center;
+padding-inline: ${(props) => props.theme.sizes.contentPadding};
+padding-block: 1.4em;
+box-shadow: ${(props) => (props.shadow === 'true' ? '0px 0.3rem 0.3rem 0px rgba(0, 0, 0, 0.05)' : 'none')};
+
+span{
+  margin: 0 auto;
+  font-weight: 700;
+  letter-spacing: -0.04638rem;
+  flex-direction: column;
+  padding-right: 11px;
+}
+
+button{
+  background: none;
+  border: none;
+}
+`;
+
+export default function Header({ children, shadow }: HeaderProps) {
+  const navigate = useNavigate();
+  return (
+    <HeaderContainer shadow={shadow}>
+      <button type="button" onClick={() => navigate(-1)} aria-label="Go Back">
```

Comment:
이런식으로 사용할 수 있도록 useNavigate 훅을 한번 래핑해서 사용하는것도 괜찮은 것 같아요~
```suggestion
      <button type="button" onClick={router.back()} aria-label="Go Back">
```